data/entities.yaml

```diff
@@ -330,6 +330,32 @@
     ignition: 0.1
     duration: [20, 40]
     product: ash
+- name: parsley
+  display:
+    attr: plant
+    char: 'p'
+  description:
+    - A fast-growing plant with a pungent aroma, often found growing near rocks.
+    - |
+      When equipped as a device, parsley enables the `read`
+      command, which can be used to turn text into values:
+      ```
+      read : Text -> (Unit + a)
+      ```
```

Comment:
Elsewhere, I see `Read -> [tyQ| Text -> a |]` so this is probably outdated, right? :)

I think it would be fair to call out the limitations of `read`, like "unlike format, you should stick to specifying type explicitly and only using it for basic types, not debugging functions."

Reply:
Good catch, I forgot to update the `parsley` description when I changed the type of `read`!  I also added an extra caveat about being unable to handle functions or delay types.